cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/DFGPass.kt

```diff
@@ -390,9 +390,16 @@ class DFGPass(ctx: TranslationContext) : ComponentPass(ctx) {
             // Unresolved call expression
             handleUnresolvedCalls(call)
         } else if (call.invokes.isNotEmpty()) {
-            call.invokes.forEach {
-                Util.attachCallParameters(it, call.arguments)
-                call.addPrevDFG(it)
+            call.invokes.forEach { funDecl ->
+                if (funDecl.isInferred && inferDfgForUnresolvedSymbols) {
+                    if (call is MemberCallExpression && !call.isStatic) {
```

Comment:
You should probably call handleUnresolvedCalls instead of copy/pasting its content here :)